README.md

```diff
@@ -17,58 +17,82 @@ option.
 
 ### Compute
 
-Computes coverage from `target/catalog.json` and `target/manifest.json` files
-found in a dbt project. You need to select documentation or test coverage by
-using the respective CLI argument.
+Compute coverage from `target/catalog.json` and `target/manifest.json` files
+found in a dbt project, e.g.
+[jaffle_shop](https://github.com/dbt-labs/jaffle_shop). You need to select
+documentation or test coverage by using the respective CLI argument.
 
 ```
-$ cd my-dbt-project
+$ cd jaffle_shop
+$ dbt run  # Materialize models
 $ dbt docs generate  # Generate catalog.json and manifest.json
 $ dbt-coverage compute doc  # Compute doc coverage, print it and write it to coverage.json file
-$ dbt-coverage compute test  # Compute test coverage
-
-# Coverage summary
-# =======================================
-# my-table-1               8/20     40.0%
-# my-table-2              20/20    100.0%
-# my-table-3               2/8      25.0%
-# =======================================
-# Total                   30/48     62.5%
+
+# Coverage report
+# =====================================================================
+# customers                                              6/7      85.7%
+# orders                                                 9/9     100.0%
+# raw_customers                                          0/3       0.0%
+# raw_orders                                             0/4       0.0%
+# raw_payments                                           0/4       0.0%
+# stg_customers                                          0/3       0.0%
+# stg_orders                                             0/4       0.0%
+# stg_payments                                           0/4       0.0%
+# =====================================================================
+# Total                                                 15/38     39.5%
+
+$ dbt-coverage compute test  # Compute test coverage, print it and write it to coverage.json file
```

Comment:
@sweco I still cannot help it, it does seem a little strange that the coverage for both tests and docs ends up being redirected into the same file 🙂

Reply:
What do you think about changing the example so that it gets written to different files?

```suggestion
$ dbt-coverage compute test --cov-report coverage-test.json  # Compute test coverage, print it and write it to coverage-test.json file
```

And the same for docs coverage and `coverage-doc.json`.

Or do you want them to be written to different files by default? I wouldn't like to have the default value dynamically generated, because it brings complexity to the CLI...

Reply:
That does indeed look a bit better @sweco.

I don't think the default needs to be automatically generated, I just thought the same file in both cases would bring confusion of a different kind 🙂

Reply:
Done here: https://github.com/slidoapp/dbt-coverage/pull/11